src/benchmarks/micro/corefx/System.Net.Http/Perf.SocketsHttpHandler.cs

```diff
@@ -20,9 +20,14 @@ namespace System.Net.Http.Tests
     public class SocketsHttpHandlerPerfTest
     {
         // the field names start with lowercase to keep to benchmark ID! do not change it
-        [Params(false, true)] public bool ssl;
```

Comment:
> Please revert the naming to lowercse, we can't rename the param names to keep the existing benchmark ids.

Is it really that important?  Seems like we can just validate that there hasn't been any meaningful change in the history, and then start over with a new name.  Being locked in to the same names forever because we're concerned about losing history seems like the wrong tradeoff, especially when you consider that one of the key goals of the perf repo is to be able to easily run tests against multiple versions, which means we can always compare two versions against each other at any moment.

Reply:
It is important. I've described that in [Microbenchmark Design Guidelines doc](https://github.com/dotnet/performance/blob/master/docs/microbenchmark-design-guidelines.md#benchmarks-are-immutable)

It's very helpful when we find a regression and want to identify the commit that have introduced it. An example from this week: https://github.com/dotnet/coreclr/issues/25726#issuecomment-511929849

Reply:
> It's very helpful when we find a regression and want to identify the commit that have introduced it.

Which is why I said "Seems like we can just validate that there hasn't been any meaningful change in the history, and then start over with a new name"